templates/mpas_analysis.bash

```diff
@@ -271,6 +271,17 @@ endYear = ${tsY2}
 startYear = ${ensoY1}
 endYear = ${ensoY2}
 
+[streamfunctionMOC]!
```

Comment:
Any reason for the exclamation mark ! at the end of lines 274-282?

Reply:
I removed the exclamation marks.